apiary-metastore-listener/src/main/java/com/expedia/apiary/extensions/metastore/listener/ApiarySnsListener.java

```diff
@@ -201,6 +218,20 @@ private void sendMessage(JSONObject json) {
     log.debug(String.format("Sending Message: {} to {}", msg, TOPIC_ARN));
     PublishResult publishResult = snsClient.publish(publishRequest);
     // TODO: check on size of message and truncation etc (this can come later if/when we add more)
-    log.debug("Published SNS Message - " + publishResult.getMessageId());
+    log.info("Published SNS Message - " + publishResult.getMessageId());
   }
+
+  private Map<String, String> getHkaasParams(Map<String, String> tableParameters) {
```

Comment:
still called hkaas

Reply:
missed it. renamed to getFilteredParams() now.